context/package.json

```diff
@@ -127,7 +128,7 @@
     "test:coverage": "jest --coverage",
     "lint": "eslint -c .eslintrc.yml --ext .js,.jsx app/static/js/",
     "lint:fix": "eslint -c .eslintrc.yml --ext .js,.jsx --fix app/static/js/",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "start-storybook -p 6006 -s app/static/public/",
```

Comment:
This may be [deprecated](https://storybook.js.org/docs/react/api/cli-options#static-dir-deprecation)?
> Starting in 6.4 the -s flag is deprecated. Instead, use a configuration object in your .storybook/main.js file. See the [images and assets documentation](https://storybook.js.org/docs/react/configure/images-and-assets#serving-static-files-via-storybook) for more information.

Also wondering ... if the point of the this is to get the msw, could you use a directory other than `public`, and then have fewer configurations to tweak, like the gitignore?